README.md

```diff
@@ -38,3 +38,6 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 [docs]: docs/README.md
 [FAQ]: docs/faq.md
 [tutorials]: docs/tutorials/README.md
+
+## Data Collection
+The software may collect information about you and your use of the software and send it to Microsoft. Microsoft may use this information to provide services and improve our products and services. You may turn off the telemetry as described in the repository. There are also some features in the software that may enable you and Microsoft to collect data from users of your applications. If you use these features, you must comply with applicable law, including providing appropriate notices to users of your applications together with a copy of Microsoft's privacy statement. Our privacy statement is located at https://go.microsoft.com/fwlink/?LinkID=824704. You can learn more about data collection and use in the help documentation and our privacy statement. Your use of the software operates as your consent to these practices.
```

Comment:
I'd link "as described in the repository" to the specific markdown file describing how to configure it. For example, to https://github.com/Azure/aks-engine/blob/master/docs/topics/telemetry.md#configuration.